spec/features/user_adds_a_study_spot_spec.rb

```diff
@@ -2,7 +2,18 @@
 
 feature "adding spot" do
   scenario "user adds a new spot" do
-    visit new_spot_path
+
+    visit new_user_registration_path
+
+    fill_in 'Email', with: 'john@example.com'
+    fill_in 'Password', with: 'password'
+    fill_in 'Password confirmation', with: 'password'
+    fill_in 'First name', with: 'John'
+    fill_in 'Last name', with: 'Example'
+    fill_in 'Username', with: 'johnex'
```

Comment:
Instead of registering a new user, use FactoryGirl to create a user and then simply sign them in. Fewer lines of code!